src/pages/documentation/migrate-to-android.md

```diff
@@ -0,0 +1,89 @@
+import Tabs from './tabs/migrate-to-android.md'
+
+# Migrate to latest Adobe Experience Platform Mobile SDK for Android
+
+If you have implemented the previous versions of the Android Mobile SDKs, then this guide will help you understand the steps required to migrate to the latest version. In summary, you will need to:
+
+1. [Switch Gradle dependencies from 1.x to 2.0](#switch-gradle-dependencies-to-latest-versions)
+2. [Update SDK initialization](#update-sdk-initialization)
+3. [Replace the deprecated API calls with the new API calls](#replace-the-deprecated-api-calls-with-the-new-api-calls1)
+
+## Switch Gradle dependencies to latest versions
+
+At this time, the following Android SDK 1.x libraries may be switched out with their respective Android SDK 2.0 libraries. See instructions on proceeding further if you have [manually imported SDK libraries](#manual-library-import) , or if you used [Gradle to manage SDK dependencies](#Gradle).
+
+<InlineAlert variant="warning" slots="text"/>
+
+Since we will not provide a `Core bundle` artifact automatically including Core, Signal, Lifecycle and Identity artifacts, you need to manually import those libraries in Android SDK 2.0.
```

Comment:
let's make this sentence impersonal
Starting with version 2.0.0, the `sdk-core` bundle (which includes Core, Lifecycle, Identity, Signal) will no longer receive updates. You need to import those libraries individually as described below.